yarn.lock

```diff
@@ -1338,6 +1338,16 @@
     lodash "^4.17.13"
     to-fast-properties "^2.0.0"
 
+"@chromaui/localtunnel@1.10.1":
```

Comment:
This dependency must have carried over from what you were doing with the Chromatic stuff.

Reply:
Ooooops! Good pickup.